editor/editor_inspector.cpp

```diff
@@ -778,8 +778,13 @@ Control *EditorProperty::make_custom_tooltip(const String &p_text) const {
 	help_bit->add_style_override("panel", get_stylebox("panel", "TooltipPanel"));
 	help_bit->get_rich_text()->set_fixed_size_to_width(360 * EDSCALE);
 
-	String text = TTR("Property:") + " [u][b]" + p_text.get_slice("::", 0) + "[/b][/u]\n";
-	text += p_text.get_slice("::", 1).strip_edges();
+	String property_name = p_text.get_slice("::", 0);
+	String text = TTR("Property:") + " [u][b]" + property_name + "[/b][/u]";
+
+	String property_doc = p_text.get_slice("::", 1).strip_edges();
```

Comment:
Might be better to properly `split()` on `::` and do the second part of the logic only if there's more than one slice?

Reply:
Done.